### PR TITLE
Release 0.0.2

### PR DESCRIPTION
**Pull request by JohanMabille**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by JohanMabille** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
 requirements:
   build:
     - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
     - cmake
     - make  # [unix]
+  host:
+    - howardhinnant_date >=3,<4
+  run:
+    - howardhinnant_date >=3,<4
```
Comment (jjerphan) [inline recipe/meta.yaml:24]: Shouldn't we make `conda-forge/howardhinnant_date-feedstock` export runtime requirements instead?

Comment (jjerphan) [reply recipe/meta.yaml:24]: Proposed by https://github.com/conda-forge/howardhinnant_date-feedstock/pull/15.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,22 +1,27 @@
 {% set name = "sparrow" %}
-{% set version = "0.0.1" %}
+{% set version = "0.0.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/xtensor-stack/sparrow/archive/{{ version }}.tar.gz
```
Comment (jjerphan) [inline recipe/meta.yaml:9]: `home` and `dev_url` in the `about` section also need to be changed.

```suggestion
  url: https://github.com/man-group/sparrow/archive/{{ version }}.tar.gz
```

File: recipe/meta.yaml (hunk 1 above)
Comment (JohanMabille) [reply recipe/meta.yaml:24]: Yes, I just wanted to avoid the availability of the dependency

**Review by jjerphan — APPROVED** [review]
LGTM.

A final re-rendering is possible.

**Comment by JohanMabille** [comment]
Yep, just waiting for the last build of Howard Hinnant to be available

**Comment by JohanMabille** [comment]
@conda-forge-admin please rerender

**Pull request merged**